local/src/index.tsx

```diff
@@ -206,6 +274,36 @@ function Editor({ initialValue }: { initialValue: Descendant[] }) {
             </Menu.Root>
           </Textbit.Gutter>
 
+          <ContextMenu.Root className='textbit-contextmenu'>
+            {!!spelling?.suggestions &&
+              <ContextMenu.Group className='textbit-contextmenu-group' key='spelling-suggestions'>
+                <>
+                  {spelling.suggestions.length === 0 &&
+                    <ContextMenu.Item
+                      className='textbit-contextmenu-item'
+                      func={() => { }}
+                    >
+                      No spelling suggestions
+                    </ContextMenu.Item>
+                  }
+                </>
+                <>
+                  {spelling.suggestions.map(suggestion => {
+                    return (
+                      <ContextMenu.Item
+                        className='textbit-contextmenu-item'
+                        key={suggestion}
+                        func={() => { alert('replacing ' + suggestion) }}
```

Comment:
Bug:
1. Right-click on a misspelled word to bring up the suggestions popover.
2. Close suggestions popover.
3. Double-click on another word to bring up the tools menu (B, I, U).
4. For me at least, the suggestions popover keeps appearing when a text formatter is selected.